Dockerfile

```diff
@@ -1,21 +1,2 @@
 FROM registry.fedoraproject.org/fedora:latest
-
-# This container image is utilized by the containers CI automation system
-# for building and testing libpod inside a container environment.
-# It is assumed that the source to be tested will overwrite $GOSRC (below)
-# at runtime.
-ENV GOPATH=/var/tmp/go
-ENV GOSRC=$GOPATH/src/github.com/containers/podman
-ENV SCRIPT_BASE=./contrib/cirrus
-ENV PACKER_BASE=$SCRIPT_BASE/packer
-
-ADD / $GOSRC
-WORKDIR $GOSRC
-
-# Re-use repositories and package setup as in VMs under CI
-RUN bash $PACKER_BASE/fedora_packaging.sh && \
-    dnf clean all && \
-    rm -rf /var/cache/dnf
-
-# Mirror steps taken under CI
-RUN bash -c 'source $GOSRC/$SCRIPT_BASE/lib.sh && install_test_configs'
+ADD /* /var/tmp/go/src/github.com/containers/podman/
```

Comment:
This makes it look like you know what the purpose of this file is. Since Dockerfiles allow comments, would you mind adding one such as "this top-level Dockerfile is used by XXX for purposes of YYY"?

Reply:
I don't really know for sure, it's only a guess that a copy of the repo. might be useful.

I s'pose the other option here is, I could delete `Dockerfile` and then we watch and see what (if anything) breaks...